lib/codeowners/cli/main.rb

```diff
@@ -65,7 +69,8 @@ def report_inconsistencies(checker)
       def create_checker(repo)
         from = options[:from]
         to = options[:to] != 'index' ? options[:to] : nil
-        checker = Codeowners::Checker.new(repo, from, to)
+        checks = Codeowners::Checker::ALL_CHECKS.select { |check| options[:"check_#{check}"] }
```

Comment:
as we have the default `true` can we validate that it's not `false`? because maybe someone will try `--check_missing_ref=false` and it will still be true right?